src/test/java/com/stripe/BaseStripeTest.java

```diff
@@ -121,6 +121,9 @@ public void setUpStripeMockUsage() {
     httpClientSpy = Mockito.spy(new HttpURLConnectionClient());
     networkSpy = Mockito.spy(new LiveStripeResponseGetter(null, httpClientSpy));
     mockClient = new StripeClient(networkSpy);
+    // The StripeClient constructor calls .setUsage on networkSpy, let's reset that so
+    // in the test cases we can just verify the calls we care about.
+    Mockito.reset(networkSpy);
```

Comment:
My worry with that, is that we would then need to make copies of all the other helper functions here that use `networkSpy` and it might be a bad experience when you're writing a test to have to choose between them and select the right one, and I hesitated to make the experience worse in 95% of cases for an edge case that only affects usage-related tests.

I was on the fence though, let me know what you prefer.